### PR TITLE
debug drag and drop issue

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: src/util/dnd.h [hunk 1]
```diff
@@ -109,13 +111,19 @@ class DragAndDropHelper {
...
+        qDebug() << "mimeData.hasText()" << mimeData.hasText() << mimeData.text();
+
         if (!mimeData.hasUrls() ||
                 (mimeData.hasText() && mimeData.text() == sourceIdentifier)) {
             return QList<QFileInfo>();
         }
 
         QList<QFileInfo> files = DragAndDropHelper::supportedTracksFromUrls(
                 mimeData.urls(), firstOnly, acceptPlaylists);
+        for (const auto file : files) {
```
Comment (uklotzde) [inline src/util/dnd.h]: Please check for Logging::debugEnabled() before executing this loop

Comment (daschuer) [reply src/util/dnd.h]: qDebug() writes always to the mixxx.log file, regardless of the logLevel. 
If we guard the loop, we don't have this info in the log file by default.

**Comment by daschuer** [comment]
Can we merge this now to 2.1? 
The plan is to use this to fix the issue in a follow up commit and decide later if we can remove these additional debug entries later.

**Comment by uklotzde** [comment]
@daschuer All builds are failing!?

**Comment by daschuer** [comment]
Ups, sorry. I have messed up my branch. Now it is working. Merge?

**Comment by daschuer** [comment]
@uklotzde Can we merge this now?

File: src/util/dnd.h [hunk 2]
```diff
@@ -141,6 +149,34 @@ class DragAndDropHelper {
...
+    static void handleTrackDragEnterEvent(QDragEnterEvent* event, const QString& group,
+                                          UserSettingsPointer pConfig) {
+        qDebug() << "handleTrackDragEnterEvent()";
+        if (allowLoadToPlayer(group, pConfig) &&
+                dragEnterAccept(*event->mimeData(), group,
+                                true, false)) {
+            qDebug() << "event->acceptProposedAction()";
+            event->acceptProposedAction();
+        } else {
+            qDebug() << "event->ignore();";
```
Comment (uklotzde) [inline src/util/dnd.h]: Please revise those log statements. At least one of them is redundant, two lines are printed on each invocation! The messages are also very low-level and only acceptable for temporary debugging, but definitely not in production code. This one also contains an unnecessary semicolon.

Two distinctive and concise messages with context would help.

**Comment by uklotzde** [comment]
Do we really need to introduce this verbose and low-level debug logging into a released version? These are actually trace logs. There should at least be a TODO to remove them after the issues have been resolved.

It's really awkward that the log level _DEBUG_ in Mixxx/Qt4 is used for _INFO_/_DEBUG_/_TRACE_ logging.

**Comment by daschuer** [comment]
I original want to have this for debug only in a single CI build. We can revert all of this if we the issue is gone, or descide which of these additions are useful.

**Comment by uklotzde** [comment]
Well, then let's just merge it. I count on you to remove those verbose and ugly log messages eventually ;)

**Pull request merged**